Missing glib2 dependency.

```
../extension-manager/src/local/shell-dbus-interface.c:17:12: fatal error: gio/gunixfdlist.h: No such file or directory
   17 | #  include <gio/gunixfdlist.h>
      |            ^~~~~~~~~~~~~~~~~~~
```

That file is part of glib2. I don't know how meson work but I'm assuming that a dependency on it is missing. More specifically what is defined in the pkg-config file `gio-unix-2.0.pc`.

--- a/src/meson.build
+++ b/src/meson.build
@@ -26,6 +26,7 @@ exm_sources = [
 exm_deps = [
   dependency('gtk4'),
   dependency('libadwaita-1'),
+  dependency('gio-unix-2.0'),
   dependency('json-glib-1.0'),
   dependency('libsoup-3.0'),
   dependency('text-engine-0.1')

